tests/conftest.py

```diff
@@ -29,7 +29,7 @@ def user():
 
 @pytest.fixture
 def admin_user():
-    return UserFactory(is_staff=True)
+    return UserFactory(is_staff=True, is_superuser=True, is_active=True)
```

Comment:
should we set `is_active=True` on the factory? We probably want users active by default, but it's likely also only checked in the admin currently.